### PR TITLE
Create overview page for the CLI

### PR DESCRIPTION
**Pull request by coverbeck**
Gives a high-level overview of the CLI.

Given the potentially uncertain future of the CLI, I didn't want to delve into all the details of what it can do, and showed how the user can discover its features.

dockstore/dockstore#4835
DOCK-2127

### CODE REVIEW

**Review by denis-yuen — APPROVED** [review]
ok in theory but build needs work

**Comment by denis-yuen** [comment]
> For example, if the PR reviewers decid this new page shouldn't exist, then we don't want to create the topic.

I don't remember what we decided, ok with whatever we wrote down. Or if we didn't maybe it should be in the PR template.

**Review by denis-yuen — APPROVED** [review]

File: docs/advanced-topics/dockstore-cli/overview.rst [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+     OR
+    --yaml <yaml file>                  Parameters to the entry in Dockstore, one map for one run, an array of maps for multiple runs (only for CWL)
+
+  Optional parameters:
+    --wdl-output-target                 Allows you to specify a remote path to provision output files to ex: s3://oicr.temp/testing-launcher/
+    --uuid                              Allows you to specify a uuid for 3rd party notifications
+    --ignore-checksums                  Allows you to ignore validating checksums of each downloaded descriptor
+
+
+Common Executed Commands
```
Comment (kathy-t) [inline docs/advanced-topics/dockstore-cli/overview.rst:124]: Common Executed Commands -> Commonly Executed Commands?

**Review by kathy-t — APPROVED** [review]

**Review by svonworl — APPROVED** [review]

**Comment by coverbeck** [comment]
Updated PR template; removed Hubflow links, see Slack [discussion](https://ucsc-gi.slack.com/archives/C05EZH3RVNY/p1704926441330179). Alternatively, we could link to our copy of Hubflow, but I don't think we want to be responsible for that.

**Review by svonworl — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+**Description**
+A description of the PR, should include a decent explanation as to why this change was needed and a decent explanation as to what this change does
+
+**Issue**
+A link to a github issue or SEAB- ticket (using that as a prefix)
+
+Please make sure that you've checked the following before submitting your pull request. Thanks!
+
+- [ ] Ensure that the PR targets the correct branch. Check the milestone or fix version of the ticket.
+- [ ] If the PR has any new pages, only **AFTER** the PR is approved by all reviewers, generate a new discourse topic, by running the action at https://github.com/dockstore/dockstore-documentation/actions/workflows/add-discourse-topic.yml. This is irreversible, which is why you want to ensure the new pages are approved before generating the topic.
```
Comment (denis-yuen) [inline .github/PULL_REQUEST_TEMPLATE.md:10]: Is it irreversible?
I'd kind of assumed that all community leads could delete topics on discourse anyway, in which case creating it before would expose any other kind of build issues with new pages

Comment (denis-yuen) [reply .github/PULL_REQUEST_TEMPLATE.md:10]: That said, clicking into the build I can see the link check and build worked
If we stick with this approach, suggesting new ticket for @ll5zh to split out just the discourse topic check into a new build so we can see at a glance the rest of the build succeeded

**Review by denis-yuen — APPROVED** [review]

**Review by kathy-t — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (coverbeck) [reply .github/PULL_REQUEST_TEMPLATE.md:10]: > Is it irreversible? I'd kind of assumed that all community leads could delete topics on discourse anyway, in which case creating it before would expose any other kind of build issues with new pages

Irreversible was too strong a word. I'll update the text.

Comment (coverbeck) [reply .github/PULL_REQUEST_TEMPLATE.md:10]: > That said, clicking into the build I can see the link check and build worked If we stick with this approach, suggesting new ticket for @ll5zh to split out just the discourse topic check into a new build so we can see at a glance the rest of the build succeeded

SEAB-6142

**Pull request merged**